catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -109,6 +133,72 @@ public void init() {
           res.header("Content-Encoding", "gzip");
         });
 
+    post(
+        "/cql/transform/:transformerId",
+        (req, res) -> {
```

Comment:
❗️ this lambda is doing a lot of things. It parses the request parameters, it executes the cql query, it constructs the response (as it goes). Let's try to extract the functionality herein into separate methods each having a single responsibility, please 😄

Reply:
This unfortunately has been the convention in all of the endpoints written with the Spark application.  While I agree with the separation of concerns suggestion here, it might be best to do this in a separate ticket, as this may ultimately be out of scope with the task at hand.

Reply:
While that's understandable, I would suggest we go ahead and refactor this as a starting point since it's fairly small. Otherwise we're just piling on tech debt (assuming there's not already a ticket to refactor this in it's entirety).

Reply:
now that i've broken down the lambda function a lot it shouldn't be difficult to translate it to a regular function